README.md

```diff
@@ -2,49 +2,84 @@
 
 <div align="center">
 
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>PORTFOLIO</b></h3>
 
 </div>
 
-
+<!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-
 - [📝 License](#license)
 
+<!-- PROJECT DESCRIPTION -->
 
+# 📖 Portfolio <a name="about-project"></a>
 
+**Portfolio** Is a website that tells about me and desplay some of my projects
 
-# 📖 Portfolio Project <a name="about-project"></a>
+## 🛠 Built With <a name="built-with"></a>
 
-> My Second Microverse Project. Setup and Mobile first
+### Tech Stack <a name="tech-stack"></a>
 
-    Portfolio : Setup and Mobile first
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
 
-## 🛠 Built With <a name="built-with"></a>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
 
-### Tech Stack <a name="tech-stack"></a>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
 
 
-- **HTML**
-- **CSS**
-- **LINTERS**
+
+### Key Features <a name="key-features"></a>
+
+- **Display the list of my projects**
+- **Display the list of my Tech Stack**
+- **Display the list of my social media links**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+

```

Comment:
[OPTIONAL]
- [ ] Kindly delete this part if you do not have a deployment link yet, once you have one you should re-implement it, doing this improves the overall professionalism and organization of your `README`.